docs/API.md

```diff
@@ -1702,10 +1703,29 @@ The above example is equivalent to the `meta/common-info` example above.
 
 **Not yet implemented**
 
-#### POST *id*/resources/name.stream
+#### GET *id*/meta/resources
```

Comment:
Would this be handled through the "meta" GET?

Regardless, there should be some mention of the "meta" GET here, no?